EventFilter/GEMRawToDigi/src/AMC13Event.cc

```diff
@@ -0,0 +1,130 @@
+#include <cstdint>
+#include "EventFilter/GEMRawToDigi/interface/AMC13Event.h"
+
+using namespace gem;
+
+uint64_t AMC13Event::getCDFHeader() const
+{
+  return (static_cast<uint64_t>(m_cb5 & 0x0f) << 60) |
```

Comment:
It would be preferable to make all these masks/offsets into descriptively named constants. (everywhere in this file)